### PR TITLE
DemoMainView

### PR DESCRIPTION
**Pull request by sanyavertolet**
This PR is a part of #2053

### What's done:
* Added view for demo browsing - an entry point to `save-demo` frontend
* Removed buggy demo dropdown from `topBarLinks`
* Added two pics for `demoMainView`

### How it looks:
screenshot

### CODE REVIEW

**Comment by akuleshov7** [comment]
I think “active demo/ creates demo” is too much

**Comment by akuleshov7** [comment]
Also usually there is a single button “create it”. It would be much more convenient

**Comment by akuleshov7** [comment]
“Set run commands” - is very confusing.
As you have already seen. Need to somehow tell user of what does he need to do, where the source file will be put and etc.

**Comment by akuleshov7** [comment]
May be even provide some transparent example (even a screenshot).

**Comment by akuleshov7** [comment]
BTW the picture with how this demo looks will also fit good to this page

**Comment by sanyavertolet** [comment]
> BTW the picture with how this demo looks will also fit good to this page (to understand user what is it). May be even some gif (where it will be animated) like GitHub does on the main page (where they input code and get result)

I have also thought of adding some kind of simple gif/movie with a demo creation example. Will need to implement something later. This PR is just a beginning as here are lots of stubs for now.

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/demo/welcome/DemoStatistics.kt [hunk 1]
```diff
@@ -0,0 +1,81 @@
+/**
```
[the remark this answers is not in the document]
Comment (sanyavertolet) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/demo/welcome/DemoStatistics.kt:1]: Agree actually, so will leave the FC implementation for now, but will remove it from the view

[the remark this answers is not in the document]
Comment (sanyavertolet) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/demo/welcome/DemoStatistics.kt:1]: So I will merge it now will keep an issue updated

**Pull request merged**